homework4/ParseTree/ParseTree/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace ParseTree;
+
+internal class Program
```

Comment:
```suggestion
public class Program
```
Думаю, из-за этого падает сборка проекта на github.